golos.emit/include/golos.emit/parameters.hpp

```diff
@@ -66,13 +67,19 @@ struct inflation_rate: parameter {
 };
 using infrate_params = param_wrapper<inflation_rate,3>;
 
-using emit_param = std::variant<infrate_params, reward_pools_param>;
+struct emit_token_symbol: parameter {
```

Comment:
propose rename it (and derived names, incl. abi) to `emit_token`, it's enough to understand it's meaning and looks better: `token_symbol.symbol.code()` vs. `token.symbol.code()`